client/data/authorizations/actions.ts

```diff
@@ -1,30 +1,62 @@
 /** @format */
+/**
+ * External Dependencies
+ */
+import { Query } from '@woocommerce/navigation';
+import { apiFetch, dispatch } from '@wordpress/data-controls';
+import { __ } from '@wordpress/i18n';
 
 /**
  * Internal Dependencies
  */
-import { Query } from '@woocommerce/navigation';
 import TYPES from './action-types';
 import {
 	AuthorizationsSummary,
 	Authorization,
+	AuthorizationsList,
 } from 'wcpay/types/authorizations';
+import { STORE_NAME } from '../constants';
+import { ApiError } from 'wcpay/types/errors';
 
 export function updateAuthorizations(
 	query: Query,
-	data: Array< Authorization >
+	authorizationsList: AuthorizationsList
```

Comment:
This will create some merge conflicts with the other PRs, but let's see how it goes 😄

Reply:
Indeed, fingers crossed XD